src/System.Management.Automation/engine/hostifaces/PowerShellProcessInstance.cs

```diff
@@ -50,38 +50,11 @@ static PowerShellProcessInstance()
         /// <param name="workingDirectory">Specifies the initial working directory for the new powershell process.</param>
```

Comment:
@PaulHigin, your last commit had 1 failures in `PowerShell-CI-linux`
Basic Job Tests.Basic tests.Can use the user specified working directory parameter with quote
```powershell
Expected exactly '/tmp/c4da1cf3-c93e-45e2-b484-87ef90eaab6c/My "Dir', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Job/Jobs.Tests.ps1: line 94
94:             $jobOutput | Should -BeExactly $path.ToString()
```